content/en/docs/installing-on-linux/on-premises/install-kubesphere-on-bare-metal.md

```diff
@@ -0,0 +1,376 @@
+---
+title: "Deploy KubeSphere on Bare Metal"
+keywords: 'Kubernetes, KubeSphere, bare-metal'
+description: 'How to install KubeSphere on bare metal.'
+
+weight: 2261
+---
+
+## Introduction
+
+In addition to the deployment on cloud, KubeSphere can also be installed on bare metal. As the virtualization layer is removed, the infrastructure overhead is drastically reduced, which brings more compute and storage resources to app deployments. As a result, hardware efficiency is improved. Refer to the example below of how to deploy KubeSphere on bare metal.
+
+## Prerequisites
+
+- Please make sure that you already know how to install KubeSphere with a multi-node cluster based on the tutorial [Multi-node Installation](../../../installing-on-linux/introduction/multioverview/).
+- Server and network redundancy in your environment.
+- Considering data persistence, for a production environment, it is recommended you prepare persistent storage and create a StorageClass in advance. For development and testing, you can use the integrated OpenEBS to provision LocalPV as the storage service directly.
+
+## Prepare Linux Hosts
+
+This tutorial uses 3 physical machines of **CentOS Linux release 7.6.1810 (Core)** with the default minimal installation. Every machine has 2 Cores, 4 GB memory and 40 G disk space.
```

Comment:
It seems the server information is missing in this guide, we need to state that what kind of server we used as the example.

![image](https://user-images.githubusercontent.com/40452856/93856440-9c6b1900-fceb-11ea-83a1-566e2952c626.png)